indigo_app/views/tasks.py

```diff
@@ -32,7 +33,25 @@ class TaskListView(TaskViewBase, ListView):
     paginate_orphans = 4
 
     def get_queryset(self):
-        return Task.objects.filter(country=self.country, locality=self.locality).order_by('-created_at')
+        try:
+            frbr_uri = self.request.GET['frbr_uri']
```

Comment:
Much like with a `dict`, you can use `self.request.GET.get('frbr_uri')` which returns None if the key doesn't exist in the dict, rather than raise an exception.